hud.c

```diff
@@ -594,6 +594,20 @@ static void Hud_EquipItemBelow(uint8 *item) {  // 8ddf00
   } while (!Hud_DoWeHaveThisItem(*item));
 }
 
+static int GetCurrentItemButtonIndex() {
+  if (enhanced_features0 & kFeatures0_SwitchLR) {
+    return (joypad1L_last & kJoypadL_X) ? 1 :
+      (joypad1L_last & kJoypadL_L) ? 2 :
+      (joypad1L_last & kJoypadL_R) ? 3 : 0;
+  }
+  return 0;
+}
+
+static uint8 *GetCurrentItemButtonPtr() {
+  int i = GetCurrentItemButtonIndex();

```

Comment:
remove this int i line to be consistent with the other copy of this function (which should be removed)